adding_data_long.qmd

```diff
@@ -1252,11 +1253,11 @@ Then in R
 ```{r, eval=FALSE}
 dataset_ids <- "Smith_1995"
 
-# Update remake file
-build_setup_pipeline()
+# Update build file
+build_setup_pipeline(method="base")
```

Comment:
FYI so "base" is the default so argument isn't needed. You need to specify if you want either remake or furrr

Reply:
Yes, but I want to specify it, especially for now since others already started using function when the default was `remake`